SimPEG/EM/Static/DC/ProblemDC_2D.py

```diff
@@ -165,6 +256,15 @@ def getSourceTerm(self, ky):
             q[:, i] = src.eval(self)
         return q
 
+    @property
+    def deleteTheseOnModelUpdate(self):
+        toDelete = []
+        if self.sigmaMap is not None or self.rhoMap is not None:
```

Comment:
Should this if statement be broken up?

Reply:
no, we should have all of the following matrices be deleted if either sigmaMap or rhoMap is not none (you might be inverting for rho, but working with MeSigma in the code)